chopper.py

```diff
@@ -373,6 +445,8 @@ def main() -> None:
     # Split by columns first to avoid full file shuffle if possible.
     if columns:
         col_list = columns.split(",")
+        ## [CG] To save a few lines you could do:
+        ##      files = [split_by_columns(f, col_list, config) for f in files]
```

Comment:
I think this doesn't work because `split_by_columns` returns a list, but I'm gonna change this to use `list(itertools.chain.from_iterable([split_by_columns(...) for f in files])).